package.json

```diff
@@ -51,7 +51,7 @@
     "moment": "^2.26.0",
     "monaco-editor": "^0.20.0",
     "next": "^11.1.3",
-    "node-fetch": "^3.1.1",
+    "node-fetch": "2.6.7",
```

Comment:
```suggestion
    "node-fetch": "^2.6.7",
```

Let's not fix the version so we can still get updates of future patches.

Reply:
Okay :)